plugins/modules/aci_config_export_policy.py

```diff
@@ -0,0 +1,312 @@
+                "name": "production"
+            }
+        }
+    }
+filter_string:
+  description: The filter string used for the request
+  returned: failure or debug
+  type: str
+  sample: ?rsp-prop-include=config-only
+method:
+  description: The HTTP method used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: POST
+response:
+  description: The HTTP response from the APIC
+  returned: failure or debug
+  type: str
+  sample: OK (30 bytes)
+status:
+  description: The HTTP status from the APIC
+  returned: failure or debug
+  type: int
+  sample: 200
+url:
+  description: The HTTP url used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: https://10.11.12.13/api/mo/uni/tn-production.json
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec, aci_annotation_spec
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(aci_annotation_spec())
+    argument_spec.update(
+        name=dict(type="str"),
+        description=dict(type="str"),
+        format=dict(type="str", choices=["json", "xml"]),
+        target_dn=dict(type="str"),
+        snapshot=dict(type="bool"),
+        export_destination=dict(type="str"),
+        scheduler=dict(type="str"),
+        start_now=dict(type="bool"),
+        state=dict(type="str", default="present", choices=["absent", "present", "query"]),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ["state", "present", ["name"]],
+            ["state", "absent", ["name"]],
+        ],
+    )
+
+    aci = ACIModule(module)
+
+    name = module.params.get("name")
+    description = module.params.get("description")
+    format = module.params.get("format")
+    target_dn = module.params.get("target_dn")
+    snapshot = aci.boolean(module.params.get("snapshot"))
+    export_destination = module.params.get("export_destination")
+    scheduler = module.params.get("scheduler")
+    start_now = aci.boolean(module.params.get("start_now"), "triggered", "untriggered")
+    state = module.params.get("state")
+
+    aci.construct_url(
+        root_class=dict(
+            aci_class="configExportP",
+            aci_rn="fabric/configexp-{0}".format(name),
+            module_object=name,
+            target_filter={"name": name},
+        ),
+        child_classes=["configRsExportScheduler", "configRsRemotePath"],
+    )
+    aci.get_existing()
+
+    if state == "present":
+        child_configs = []
+        if scheduler is not None:
+            child_configs.append(
+                dict(
+                    configRsExportScheduler=dict(
+                        attributes=dict(tnTrigSchedPName=scheduler),
+                    )
+                )
+            )
+        if export_destination is not None:
```

Comment:
can you add test case for export_destination?

Reply:
test cases updated